### ISSUE TITLE
Picking up an object reorients it instantaneously, which seems like a glitch

### ISSUE THREAD

**Issue by ara4n** [matrix-org/thirdroom#269]
* Throw a bouncy ball
* Pick it up
* Observe that its orientation suddenly snaps to some alignment (presumably Z is up or whatever)

I'd expect picking something up to preserve its orientation.

**Comment by ara4n** [comment]
(and possibly preserve its angular momentum too)

**Comment by vintprox** [comment]
Yes! It's such a disruptive mechanic - out of nowhere, forces appear to get object in unnatural position and angle, always facing player like a stalker. And of course, it moves everything else with it. You can't build anything viable, because your Jenga tower will most likely fall.

Once relative position and angle are preserved during and after grabbing, user will have a better control of positioning the props.
